client/blocks/product-purchase-features/product-purchase-features-list/jetpack-return-to-dashboard.jsx

```diff
@@ -21,7 +21,7 @@ export default localize( ( { selectedSite, translate } ) => {
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
 				icon="house"
-				title={ translate( 'Return to your site\'s dashboard' ) }
+				title={ translate( 'Return to Your Dashboard' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Return to your site's dashboard' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jetpack-myplan-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).